src/usr/local/buildpack/tools/dotnet.sh

```diff
@@ -10,7 +10,7 @@ if [[ ! "${MAJOR}" || ! "${MINOR}" || ! "${PATCH}" ]]; then
   exit 1
 fi
 
-DOTNET_INSTALL_DIR=/usr/local/dotnet/${TOOL_VERSION}
+DOTNET_INSTALL_DIR=/usr/local/${TOOL_NAME}/${TOOL_VERSION}
```

Comment:
Is there a scenario where this is not `dotnet` ?

Reply:
not yet, just some simplification, so i can easily copy'n'past between tools 🙃